modules/RTC/RTCUtils.js

```diff
@@ -1287,167 +1287,142 @@ class RTCUtils extends Listenable {
      * @returns {*} Promise object that will receive the new JitsiTracks
      */
     obtainAudioAndVideoPermissions(options = {}) {
-        const self = this;
-
         const dsOptions = {
```

Comment:
`ds` = `desktopSharing`? i think it'd be clearer if we changed this to be `desktopSharingOptions`